__tests__/app/api/friends/accept/route.test.ts

```diff
@@ -197,13 +202,95 @@ describe('/api/friends/accept', () => {
     });
 });
 
+describe('calls to pusher',()=>{
+    let triggerSpy: jest.SpyInstance;
+
+    beforeEach(() => {
+        triggerSpy = jest.fn();
+        (getPusherServer as jest.Mock).mockReturnValue({trigger: triggerSpy});
+        (fetchRedis as jest.Mock).mockImplementation(redisMock());
+    })
+
+    afterEach(()=>{
+        jest.resetAllMocks();
+    })
+
+    test('id to add is 12345 expect pusher.trigger to be called with first arg "user__12345__friends"', async ()=>{
+        fetchMock.mockResponseOnce(JSON.stringify({ success: true }));
+        (myGetServerSession as jest.Mock).mockResolvedValue({user:{id:'1966'}});
+        const req = requestFromId('12345')
+        await POST(req);
+        expect(triggerSpy).toHaveBeenCalledWith("user__12345__friends", expect.anything(), expect.anything());
+    })
+
+    test('id to add is 54321 expect pusher.trigger to be called with first arg "user__12345__friends"', async ()=>{
+        fetchMock.mockResponseOnce(JSON.stringify({ success: true }));
+        (myGetServerSession as jest.Mock).mockResolvedValue({user:{id:'1966'}});
+        const req = requestFromId('54321')
+        await POST(req);
+        expect(triggerSpy).toHaveBeenCalledWith("user__54321__friends", expect.anything(), expect.anything());
+    })
+
+    test('id to add is 54321 expect pusher.trigger to be called with event "new_friend"', async ()=>{
+        fetchMock.mockResponseOnce(JSON.stringify({ success: true }));
+        (myGetServerSession as jest.Mock).mockResolvedValue({user:{id:'1966'}});
+        const req = requestFromId('54321')
+        await POST(req);
+        expect(triggerSpy).toHaveBeenCalledWith(expect.anything(), 'new_friend', expect.anything());
+    })
+
+    test('expect pusher.trigger to be called with data of the current user', async ()=>{
+        fetchMock.mockResponseOnce(JSON.stringify({ success: true }));
+        (myGetServerSession as jest.Mock).mockResolvedValue({user:{id:'1966'}});
+        const req = requestFromId('54321')
+        await POST(req);
+        expect(triggerSpy).toHaveBeenCalledWith(expect.anything(), expect.anything(), {
+            name: 'Adam',
+            email: 'adam@batcave.com',
+            image: 'stub',
+            id: '1966'
+        });
+    })
+    test('expect pusher.trigger to be called with data of the current user, different data', async ()=>{
+        fetchMock.mockResponseOnce(JSON.stringify({ success: true }));
+        (myGetServerSession as jest.Mock).mockResolvedValue({user:{id:'1701'}});
+        const req = requestFromId('54321')
+        await POST(req);
+        expect(triggerSpy).toHaveBeenCalledWith(expect.anything(), expect.anything(), {
+            name: 'William',
+            email: 'bill@canda.ca',
+            image: 'stub',
+            id: '1701'
+        });
+    })
+
+    test('needs two calls: one for the user and one for the id to add', async ()=>{
```

Comment:
These tests are thorough, but they may be on implementation instead of behavior. Would some kind of E2E test be useful here to confirm the live update behavior?